### PR TITLE
roboirc's PR and modifications

### PR DESCRIPTION
**Pull request by roboirc**

### CODE REVIEW

**Comment by AndrioCelos** [comment]
What's the purpose of this pull request? What changes are you proposing?

**Comment by roboirc** [comment]
The purpose of this pull request is to include better tool tips within KVIrc. I am hoping to improve the tool tips in KVIrc so the software's features are clearer for others via tool tips.

**Comment by wodim** [comment]
The only change I like is the one in KviIrcView.cpp

**Comment by roboirc** [comment]
Ok. Hopefully more coming in future.

**Comment by roboirc** [comment]
How does PR work? Who includes these changes if they are accepted?

File: src/kvirc/ui/KviChannelWindow.cpp [hunk 1]
```diff
@@ -104,7 +104,7 @@ KviChannelWindow::KviChannelWindow(KviConsoleWindow * lpConsole, const QString &
 	    this, SLOT(topicSelected(const QString &)));
 	// mode label follows the topic widget
 	m_pModeWidget = new KviModeWidget(m_pTopSplitter, *this, "mode_");
-	KviTalToolTip::add(m_pModeWidget, __tr2qs("Channel modes"));
+	KviTalToolTip::add(m_pModeWidget, __tr2qs("Channel settings button."));
```
Comment (AndrioCelos) [inline src/kvirc/ui/KviChannelWindow.cpp]: This is the channel modes text box, not a button. That text is rarely seen anyway unless you're on a channel with no modes.

File: src/kvirc/ui/KviInput.cpp [hunk 2]
```diff
@@ -87,6 +87,8 @@ KviInput::KviInput(KviWindow * pPar, KviUserListView * pView)
 	m_pHideToolsButton->setIconSize(QSize(22, 22));
 	m_pHideToolsButton->setFixedWidth(16);
 
+	m_pHideToolsButton->setToolTip("Click here to show/hide various input settings");
```
Comment (DarthGandalf) [inline src/kvirc/ui/KviInput.cpp]: Did you miss the `__tr2qs`? The message needs to be translateable

**Comment by DarthGandalf** [comment]
The commit messages need a better description

File: src/kvirc/ui/KviInput.cpp (hunk 2 above)
Comment (roboirc) [reply src/kvirc/ui/KviInput.cpp]: Updated it.

**Comment by roboirc** [comment]
What did the check fail on? Travis CI?

**Comment by ctrlaltca** [comment]
Looks like it failed because a job failed to start on Travis. I've just restarted it and it worked: https://travis-ci.org/kvirc/KVIrc/jobs/612232346
Please have a look at other devs' comments

**Comment by roboirc** [comment]
Thanks will do.

**Comment by roboirc** [comment]
Now tooltip for icons in Icon Table shows name of icon which is important when using them. Before that info was not visible.

**Comment by roboirc** [comment]
@ctrlaltca how come CI is failing again?

**Comment by ctrlaltca** [comment]
Somethign about perl support on windows, unrelated to your change.

**Comment by roboirc** [comment]
Ok cool. How come it shows X next to ac9f957? The change has not yet been accepted or failed CI again or what?

**Comment by roboirc** [comment]
Updating PR with more changes as follows. All changes by me are indicated by my nickname in comments:
- Increased size of icons in Icon Manager with centered window
- Before the option KviOption_boolCommandlineInUserFriendlyModeByDefault did not work properly, now it either enables or disables the friendly mode based on the option in menu. 
- You can open destination folder for downloaded files via right click popup menu on the item.
- Colour tint on filename after right clicking on a downloaded file has been removed as it makes filename hard to see.

To download specific commits look under Commits tab in this git. And on the right side you can click on the <> symbol to download that specific git.

**Comment by staticfox** [comment]
1) Please remove the cmake build artifacts
2) Please remove the autogenerated docs
3) Please remove your IDE's config stuff

I can't even look at the proper changes since GitHub truncates the files changed. It appears that the intent of this PR involves changes that are unrelated to each other, so perhaps you could open a PR for each feature/bug/enchangement fix you are proposing.

**Comment by roboirc** [comment]
@staticfox By 2, you mean the KVIrc/doc folder?

**Comment by roboirc** [comment]
Dear staticfox, could you please guide me on how to remove those 3? thanks

**Comment by roboirc** [comment]
How do I update this PR with latest changes? I am unable to update this PR any more. Thanks.

**Comment by ctrlaltca** [comment]
I'm sorry but i have to close this PR. It's just impossible to be reviewed.

**Pull request closed without merging**